### ISSUE TITLE
pretrained vgg weight

### ISSUE THREAD

**Issue by LeeDongYeun** [wtjiang98/BeautyGAN_pytorch#6]
Could you upload the pre-trained vgg weight or let me know how to get the weight?

**Comment by LeeDongYeun** [comment]
I think we can find the pretrained loss here.
https://github.com/SunnerLi/P-Conv/issues/1
